assets/sass/_fonts.scss

```diff
@@ -0,0 +1,16 @@
+// Each font referenced here should be preloaded in for-head.mustache
+
+/* Demi */
+@font-face {
+  font-display: fallback;
+  font-family: "Avenir Next Webfont";
+  src: url("../fonts/249228f0-61ac-40cc-a5a5-5609c9816e3f.woff2") format("woff2");
```

Comment:
Where did the font name come from? Since we're caching busting elsewhere, could be something easier to work with?